src/main/webapp/i18n/de/showStatistic.json

```diff
@@ -38,8 +38,8 @@
         "dragAndDropQuestionStatistic": {
             "xAxes": "DropLocation",
             "yAxes": "Häufigkeit",
-            "showSampleSolution": "Zeige Beispiellösung",
-            "hideSampleSolution": "Verberge Beispiellösung"
+            "showSampleSolution": "Zeige Beispiel Lösung",
```

Comment:
`Lösungsbeispiel` or `Beispiellösung` would be a better wording

Reply:
I am fine with either, however we have to be consistent in the whole user interface.